frontend/src/components/organizations/OrganizationInfo.vue

```diff
@@ -242,45 +240,50 @@ export default {
       uploadedDocumentsEdit: [],
       documentsToUpload: [],
       documentsToDelete: [],
-      showUploadDocumentsAlert: false,
       loadingInclusionPolicy: false,
     }
   },
-  async updated() {
+
+  computed: {
+    documentsExistOrBeingAdded() {
```

Comment:
Technically this doesn't need to be in computed as it's not referenced in the template. But it's also fine as is if you prefer to split this logic out.

Reply:
let's keep it. I find it is easier to read this way.